openmicroscopy/webtest
# Static demo viewer

Static viewer page to test bugs reported in
https://www.openmicroscopy.org/community/viewtopic.php?f=6&t=8058

Previously opened as #10 with robot tests, but these have been removed now for simplicity.
Robot framework testing of OMERO.web apps is a separate discussion.

To test manually:

Go to http://web-dev-merge.openmicroscopy.org/webtest/static_example/ (no need to log in)

 - Check that using the +, - and 1:1 buttons on each viewer ONLY zooms that viewer (and not both viewers) Fixed in openmicroscopy/openmicroscopy#4701
 - Test that clicking thumbnails of 'Big' images then 'small' images loads them into viewport and viewport works for both. Fixed in openmicroscopy/openmicroscopy#4695
 - With smaller images selected, enable the "Sync zoom" and/or "Sync centre" checkboxes in far top-left of page. Now, the viewers should zoom in together and/or pan around to the same centre-point.
